packages/api/README.md

```diff
@@ -31,20 +31,28 @@ Log into a server or create accounts using these APIs. You'll need an active ses
 
 ```typescript
 import { BskyAgent, AtpSessionEvent, AtpSessionData } from '@atproto/api'
+
+// configure connection to the server, without account authentication
 const agent = new BskyAgent({
   service: 'https://example.com',
   persistSession: (evt: AtpSessionEvent, sess?: AtpSessionData) => {
     // store the session-data for reuse
   },
 })
 
-await agent.login({ identifier: 'alice@mail.com', password: 'hunter2' })
-await agent.resumeSession(savedSessionData)
+// create a new account on the server
 await agent.createAccount({
   email: 'alice@mail.com',
   password: 'hunter2',
   handle: 'alice.example.com',
+  inviteCode: 'someCode',
 })
+
+// if an existing session (accessed with 'agent.session') was securely stored previously, then reuse that
```

Comment:
Hi, @bnewbold How to check whether there is an existing session already?x

Reply:
I think that in short, you either have session information out of band (eg, your program/bot/client wrote it to disk and re-loaded it), or not. It will depend on your program.